sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient/FormContentType.cs

```diff
@@ -11,6 +13,7 @@ public enum FormContentType
         /// <summary>
         /// Used for JSON files.
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         Json,
```

Comment:
`Json` documents are not supported by the service. This was added in the convenience layer by mistake. It's been out for years at this point and we never got feedback about it so it's unlikely that customers are actually struggling trying to use it. We're hiding it here just to be safe.